bridge/hardhat.config.js

```diff
@@ -226,7 +227,7 @@ function getMultiSigAddressesByChainId() {
   const multiSigAddressesByChainId = {};
   multiSigAddressesByChainId[chains.ETHEREUM_MAIN_NET_CHAIN_ID] = '0x040007b1804ad78a97f541bebed377dcb60e4138';
   multiSigAddressesByChainId[chains.RSK_MAIN_NET_CHAIN_ID] = '0x040007b1804ad78a97f541bebed377dcb60e4138';
-  // TODO: add two level mapping to allow rsk testnet to be used from multiple external testnet chains.
+  // add two level mapping to allow rsk testnet to be used from multiple external testnet chains.
```

Comment:
this is what you're doing right? I'd maybe remove the comment altogether

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/276/commits/169edba6f20f38f122100142fc48c2b8ebab5f2c